### PR TITLE
Added 1018 prerelease in WebView2 relnotes

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

**Comment by peiche-jessica** [comment]
I'd use `[Webview2ReferenceWin32Icorewebview2experimental9ViewWebview2101018Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimental9?view=webview2-1.0.1018-prerelease&preserve-view=true "interface ICoreWebView2Experimental9 | Microsoft Docs"` for media apis and `[Webview2ReferenceWin32Icorewebview2experimentalenvironment8ViewWebview2101018Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironment8?view=webview2-1.0.1018-prerelease&preserve-view=true "interface ICoreWebView2ExperimentalEnvironment8 | Microsoft Docs"` for profile apis.

**Comment by peiche-jessica** [comment]
Are there any bugs fixed between 1010-1018?

**Comment by mikehoffms** [comment]
I guess I won't change the Stage of this PR to Draft, because the stages are called "Draft" or "Ready for Review", and this PR is ready for review.  Debatable.

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -1121,6 +1139,9 @@ Initial developer preview release.
 [GithubMicrosoftedgeWebviewfeedbackIssue411]: https://github.com/MicrosoftEdge/WebViewFeedback/issues/411 "Feedback repo for MicrosoftEdge/WebViewFeedback Issue 411"
 [GithubMicrosoftedgeWebviewfeedbackIssue238]: https://github.com/MicrosoftEdge/WebViewFeedback/issues/238 "Feedback repo for MicrosoftEdge/WebViewFeedback Issue 238"
 [GithubMicrosoftedgeWebviewfeedbackIssue1077]: https://github.com/MicrosoftEdge/WebViewFeedback/issues/1077 "Feedback repo for MicrosoftEdge/WebViewFeedback Issue 1077"
+[GithubMicrosoftedgeWebviewfeedbackIssue1723]: https://github.com/MicrosoftEdge/WebViewFeedback/issues/1723 "Feedback repo for MicrosoftEdge/WebViewFeedback Issue 1723"
+[GithubMicrosoftedgeWebviewfeedbackIssue730]: https://github.com/MicrosoftEdge/WebViewFeedback/issues/730 "Feedback repo for MicrosoftEdge/WebViewFeedback Issue 730"
+[GithubMicrosoftedgeWebviewfeedbackIssue1528]: https://github.com/MicrosoftEdge/WebViewFeedback/issues/1528 "Feedback repo for MicrosoftEdge/WebViewFeedback Issue 1528"
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:1144]:
```suggestion
[GithubMicrosoftedgeWebviewfeedbackIssue1548]: https://github.com/MicrosoftEdge/WebViewFeedback/issues/1548 "Feedback repo for MicrosoftEdge/WebViewFeedback Issue 1548"
```
I'm fixing this locally now.  1548 at top of article is correct, 1528 is wrong

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -26,6 +26,24 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+Release Date: September 20, 2021
+
+[NuGet package for WebView2 SDK 1.0.1018-prerelease][NuGetGallery1.0.1018-prerelease]
+
+For full API compatibility, this prerelease version of the WebView2 SDK requires Microsoft Edge version 95.0.1018.0 or higher.
+
+### General
+
+#### Experimental Features
+*   Added [Media API][Webview2ReferenceWin32Icorewebview2experimental9ViewWebview2101018Prerelease] that enables developers to mute/unmute media within the webview.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:40]: Are the links on these two lines going to be revised to go to mid-page sections?  The page that I arrive at (after deleting "review." from the 404 URL)  doesn't have a clear relationship to the link text here.
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1468#experimental-features

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor pass done. 
Entered a comment asking to clarify the Media API link or destination; expect 'media' in the destination page.

**Comment by mikehoffms** [comment]
Removed "Draft" stage, returned the PR to "Ready for review" stage, because the NuGet destination is live.

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -27,6 +27,25 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
...
+Release Date: September 20, 2021
+
+[NuGet package for WebView2 SDK 1.0.1018-prerelease][NuGetGallery1.0.1018-prerelease]
+
+For full API compatibility, this prerelease version of the WebView2 SDK requires Microsoft Edge version 95.0.1018.0 or higher.
+
+### General
+
+#### Experimental Features
+*   Added [Media API][Webview2ReferenceWin32Icorewebview2experimental9ViewWebview2101018Prerelease] that enables developers to mute/unmute media within the WebView.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:41]: @jasonstephen15 - I'm re-entering this "Outdated" comment, for visibility in the GitHub UI.

The destination page lacks the word 'media'. Will this "Media API" link be revised to go to a mid-page section that will have the word 'media'? The page that I arrive at doesn't have a clear relationship to the link text here.
The present .md file, rendered:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-1468#experimental-features
(To test the Media API link, delete "review." from the 404 URL after click "Media API".)

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/release-notes.md]: @mikehoffms The API members on ICoreWebView2Experimental9 interface are various APIs to provide media control such as `IsMuted` and `IsDocumentPlayingAudio` properties. I believe "media" here is a generic term. Would `[media APIs](link)` be better in this context?

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -1245,4 +1268,6 @@ Initial developer preview release.
 [Webview2ReferenceWin32ICoreWebView2ExperimentalSettings6ViewWebview210999Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalsettings6?view=webview2-1.0.1010-prerelease&preserve-view=true "interface ICoreWebView2ExperimentalSettings6 | Microsoft Docs"
 [Webview2ReferenceWin32ICoreWebView2ExperimentalCompositionController3ViewWebview210999Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalcompositioncontroller3?view=webview2-1.0.1010-prerelease&preserve-view=true "interface ICoreWebView2ExperimentalCompositionController3 | Microsoft Docs"
 
+[Webview2ReferenceWin32Icorewebview2experimental9ViewWebview2101018Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimental9?view=webview2-1.0.1018-prerelease&preserve-view=true "interface ICoreWebView2Experimental9 | Microsoft Docs"
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:1271]:
```suggestion
[Webview2ReferenceWin32Icorewebview2experimental9ViewWebview2101018Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimental9?view=webview2-1.0.1018-prerelease&preserve-view=true#summary "interface ICoreWebView2Experimental9 | Microsoft Docs"
```

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: Lowercased 'media' and linked to Summary # section which is clearly media-related unlike top of Ref page.

**Review by mikehoffms — APPROVED** [review]

**Comment by mikehoffms** [comment]
I Approved the PR, but I didn't attempt to add info per Jessica in a comment above.

**Comment by jasonstephen15** [comment]
@mikehoffms - good to merge!

**Pull request merged**